### ISSUE TITLE
Question: Is it possible to display conversation threads? |  | 質問：会話のスレッドを表示することは可能ですか？

### ISSUE THREAD

**Issue by rafo** [arrow2nd/nekome#128]
Currently no conversation threads (As described in https://developer.twitter.com/en/docs/twitter-api/conversation-id) are displayed, right? If not, will it be possible soon?

---

現在、会話スレッドは表示されていませんね。
https://developer.twitter.com/en/docs/twitter-api/conversation-id に記載されているように？そうでない場合、すぐに可能になるのでしょうか？

**Comment by arrow2nd** [comment]
Thank you for using nekome 😻
Also, thank you very much for your kind translation.

Regarding the display of tweet threads, this is one of the features I would like to implement in the future (#108).

However, due to a change in Twitter's developer policy that prohibits the development of third-party clients, I cannot promise the implementation of this feature.

I am sorry....

<details>
<summary>Japanese</summary>
nekomeを使ってくださりありがとうございます 😻

また、ご丁寧な翻訳もありがとうございます。

ツイートスレッドの表示についてですが、これは将来的に実装したい機能の1つです（#108）

しかし、Twitter社の開発者ポリシーの変更によって、サードパーティクライアントの開発が禁止されてしまったため、この機能を実装するかどうかお約束することはできません。

申し訳ないです…
</details>

**Comment by rafo** [comment]
That's right, there was a change. Thanks for your answer! I hope your CLI client will be developed further. It would be a pity if there would be no more possibility for twitter in the terminal. 

---

そうです、変化があったんです。ご回答ありがとうございました。CLIクライアントがさらに発展することを期待しています。端末でtwitterができる可能性がなくなってしまうのは、とても残念なことです。
